stubs/python-jose/jose/jwt.pyi

```diff
@@ -6,14 +6,14 @@ from .backends.base import Key
 def encode(
     claims: MutableMapping[str, Any],
     # Internally it calls jws.sign() that expects a key dict instance instead of Mapping
-    key: str | dict[str, Any] | Key,
+    key: str | bytes | dict[str, Any] | Key,
     algorithm: str = ...,
     headers: Mapping[str, Any] | None = ...,
     access_token: str | None = ...,
 ) -> str: ...
 def decode(
-    token: str,
-    key: str | Mapping[str, Any] | Key,
+    token: str | bytes,
```

Comment:
It is passed down to `jws.verify()`, which accepts both `str` and `bytes` https://github.com/python/typeshed/pull/8258/files#r916986297